Marlin/planner.cpp

```diff
@@ -932,6 +932,14 @@ void Planner::_buffer_line(const float &a, const float &b, const float &c, const
       }
     #endif
   #endif
+  
+  #if ENABLED(BLOCK_LCD_ON_SHORT_MOVES)
+    #if ENABLED(SLOWDOWN)
+      block->segment_time = segment_time;
+    #else
+      block->segment_time = lround(1000000.0/inverse_mm_s);
+    #endif
```

Comment:
This is fine. But if you're a stickler for eliminating redundancies like I am, this is also fine:
```cpp
   block->segment_time =
     #if ENABLED(SLOWDOWN)
        segment_time
      #else
        lround(1000000.0 / inverse_mm_s)
      #endif
    ;
```